src/main/java/edu/harvard/iq/dataverse/DatasetFieldValue.java

```diff
@@ -122,7 +122,7 @@ public String getUnsanitizedDisplayValue() {
             if (StringUtils.isBlank(format)) {
                 format = "#VALUE";
             }           
-            String value = this.getValue();    
+            String value = this.getDisplayValue();    
```

Comment:
@tcoupin - this doesn't seem related to the issue - is it solving a different problem or is it not needed? (FYI - I'm working on a few i18n issues at the moment and am making changes to getDisplayValue and related calls so I want to make sure I understand what the issue is here.)

Reply:
@tcoupin - checking again - this looks like it calls the getDisplayValue() method which will sanitize the HTML as part of the getUnsanitizedDisplayValue() method which seems odd. I haven't looked through the call chain but presumably the callers want unsanitized values here and if that's a problem, perhaps the calling methods should use getDisplayValue instead of getUnsanitizedDisplayValue instead of making this change?

Reply:
Hi @qqmyers ,
sorry for my late response. I made this fix for 4.20 so I don't remember why I made this change bu I think it's related to dropdown list https://github.com/IQSS/dataverse/blob/a875db778a53db3353e6b3095e52bdbe6fff01e9/src/main/java/edu/harvard/iq/dataverse/DatasetField.java#L323

Reply:
@tcoupin - since a lot has changed since 4.20, can you drop that line from the PR and we can get the other change merged? I'm definitely concerned about calling the method to sanitize the values inside the getUnsanitizedDisplayValue method  - it might have fixed a 4.20 issue but presumably something that needs unsanitized values would break with this change. If it is still needed to fix a bug in 5.x, it may make more sense to, for example, change line 323 in your comment above rather than changing the getUnsanitizedDisplayValue method itself.

Reply:
I remove the change on line 125 and it's still working.
I've just rebased on develop and pushed my commit

Reply:
>  I've just rebased on develop and pushed my commit

Just a quick comment that I think we could learn from @tcoupin about rebasing and keeping a clean git history. It's nice that this pull request is just a single commit with only the change we want and no merge commits. Thanks!